src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.ts

```diff
@@ -28,6 +28,11 @@ import { onError } from 'app/shared/util/global.utils';
 import { AuxiliaryRepository } from 'app/entities/programming-exercise-auxiliary-repository-model';
 import { SubmissionPolicyType } from 'app/entities/submission-policy.model';
 
+export interface Reason {
```

Comment:
[nit] I think this interface is already existing. Is it possible to reuse it?

Reply:
Outsource of Reason and reuse implemented in 6817b0d.